example/android/build.gradle

```diff
@@ -20,14 +8,20 @@ allprojects {
     }
 }
 
-rootProject.buildDir = '../build'
+rootProject.layout.buildDirectory = '../build'
+
 subprojects {
-    project.buildDir = "${rootProject.buildDir}/${project.name}"
+    project.layout.buildDirectory = "${rootProject.layout.buildDirectory}/${project.name}"
+    if (project.hasProperty("android")) {
+        if (project.android.hasProperty("namespace")) {
+            namespace 'com.clevertap.clevertap_plugin'
```

Comment:
This namespace doesn't belong here? It is for the clevertap plugin and not the example app

Reply:
done.